services/ui-src/src/components/fields/TextField.tsx

```diff
@@ -28,14 +28,14 @@ export const TextField = ({
   form.register(name);
 
   // set initial display value to form state field value or hydration value
-  const hydrationValue = props?.hydrate;
+  const hydrationValue = props?.hydrate || defaultValue;
   useEffect(() => {
     // if form state has value for field, set as display value
     const fieldValue = form.getValues(name);
-    if (fieldValue || fieldValue == "") {
```

Comment:
Note: I don't see any reason why we're doing this, especially since the other string-based fields don't use it. Can we remove it?